.circleci/main.yml

```diff
@@ -1697,6 +1726,11 @@ workflows:
           requires:
             - jupyter-extension-build
             - build-pachctl-bin
+      - test-python-sdk:
```

Comment:
This job should get copied into the `release` workflow

Reply:
Are these not in the release workflow?

Reply:
oh my bad!